src/puptoo/process/profile.py

```diff
@@ -644,6 +650,42 @@ def system_profile(
     if gcp_network_interfaces:
         profile["public_ipv4_addresses"] = _remove_empty_string(gcp_network_interfaces.public_ips)
 
+    if iris_cpfs and iris_list:
+        try:
+            iris_profile = {}
+            iris_profile["iris_system"] = True
```

Comment:
Do we really need this field - `iris_system` Just represent the system is InterSysytems? IMO, checking if the array i.e. `iris_running_instances` is not empty then also we can conclude the same. Any thoughts?

Reply:
Okay, then I think we are good. 
Do we need to change the naming of those fields as per the JIRA card?

Reply:
Yep, I will update accordingly after confirming the detail changes in [Jira](https://issues.redhat.com/browse/RHINENG-7061) .